agents-api/tests/test_secrets_queries.py

```diff
@@ -14,10 +14,39 @@
 from tests.fixtures import pg_dsn, test_developer_id
 
 
+async def clean_secrets(developer_id, connection_pool):
```

Comment:
I believe you have to convert this to a fixture and make it clean secrets on setup and teardown